### PR TITLE
Note for how to retrieve recorded metrics

### PR DESCRIPTION
**Pull request by igrigorik**
Re: https://lists.w3.org/Archives/Public/public-web-perf/2016May/0024.html

### CODE REVIEW

**Comment by igrigorik** [comment]
@plehegar @toddreifsteck can you guys do a quick review and merge if it looks good?

File: index.html [hunk 1]
```diff
@@ -148,6 +148,13 @@
         &lt;/body&gt;
       &lt;/html&gt;
       </pre>
+      <p class="note">[[PERFORMANCE-TIMELINE]] defines two mechanisms that
+      can be used to retrieve recorded metrics: <code>getEntriesByType()</code>
```
Comment (toddreifsteck) [inline index.html]: getEntries and getEntriesByName can also be used.

**Comment by toddreifsteck** [comment]
Minor feedback. LGTM otherwise.

**Comment by igrigorik** [comment]
Good catch, thanks Todd! Updated, merging.

**Pull request merged**